app/helpers/fhir_helper.rb

```diff
@@ -249,7 +261,8 @@ def close_contact_as_fhir(close_contact)
         to_positive_integer_extension(close_contact.assigned_user, 'assigned-user'),
         to_unsigned_integer_extension(close_contact.contact_attempts, 'contact-attempts'),
         to_string_extension(close_contact.notes, 'notes'),
-        to_reference_extension(close_contact.enrolled_id, 'Patient', 'enrolled-patient')
+        to_reference_extension(close_contact.enrolled_id, 'Patient', 'enrolled-patient'),
+        to_datetime_extension(close_contact.created_at, 'created-at')
```

Comment:
Thoughts on capturing this as a `contained` `Provenance` resource instead like we do with patient?

Ditto for the created-at fields added to vaccines, labs, and assessments.

Reply:
Yeah so that's a tricky question that I spent some time wrestling with. The problem I was running into was that [Provenance](https://www.hl7.org/fhir/provenance.html) requires an [agent](https://www.hl7.org/fhir/provenance-definitions.html#Provenance.agent), and in this case it isn't exactly clear who the agent would be. We do not track the users who create a resource for these resources, unlike in the case of the Patient.

Reply:
Let me know if you have thoughts about how we could populate that `agent` value. If we can figure out a good way to do it, I can definitely move things over to `Provenance`, since I do like that consistency with `Patient`.

Reply:
Perhaps assigned user on the patient, but that's not guaranteed to exist. I don't think we'll be able to make this work unless we also start recording created by user on these associated models (which we should do anyways).

Reply:
Ok, so as of https://github.com/SaraAlert/SaraAlert/pull/1016/commits/32602e5574a2690b5100fe9319363505e2faf485 I haven't made any changes yet, since I'm not sure there is a clear path forward on this at this point. I think @holmesie's suggestion of recording the creator is the ideal, but I'm also not sure it makes sense to add that as a part of this PR, when we'd really just be adding that to make our FHIR feel neater, and not because people are necessarily looking for that info.

Reply:
Yeah, agreed that would be out of scope here, but would be a good future consideration. Just trying to limit the number of extensions we use, but it is necessary here.